server/game/cards/07-WotW/IuchiDaiyu.js

```diff
@@ -0,0 +1,21 @@
+const DrawCard = require('../../drawcard.js');
+
+class IuchiDaiyu extends DrawCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: '+1 military for each faceup province',
+            condition: () => this.game.isDuringConflict(),
```

Comment:
This should probably check for an opponent too.

Alternatively, you could make the helper function return the number of faceup provinces their opponent controls (0 if no opponent)

Reply:
I have added a new function `getNumberOfOpponentsFaceupProvinces(predicate)` that will return 0 if opponent is not defined.